lib/Models/CatalogItem.js

```diff
@@ -320,13 +325,80 @@ var CatalogItem = function(terria) {
         }
     });
 
+    /**
+     * Gets the CatalogItems current time.
+     *
+     * This property is an observable version of clock.currentTime, they will always have the same value.
+     *
+     * This property is effectively an interface adapter for clock.definitionChanged which changes the structure from
+     * an Event when the current time changes to a knockout property which can be observed.
+     *
+     * @member {JulianDate} The CatalogItems current time.
+     * @memberOf CatalogItem.prototype
+     */
+    knockout.defineProperty(this, 'currentTime', {
+        // This is defined as a property rather then a member to restrict acess to set. Clients can always
+        // `item.clock.currentTime = value` if they need to set, but since this is a copy set would need to copy the
+        // change to item.clock.currentTime.
+        get : function() {
+            return this._currentTime;
+        },
+    });
+
+    /**
+     * Gets the CatalogItems current time as the discrete time that time the catalogItem has information for.
```

Comment:
Extra word?  `time that time the`.

Reply:
Done 033d2275b31386248dc665e528ab15575cf2232c